app/assets/stylesheets/common/base/login.scss

```diff
@@ -5,6 +5,10 @@ $invite-mobile-bg: absolute-image-url("/branded-background-mobile.svg");
   color: var(--danger);
   font-size: var(--font-down-1);
   font-weight: bold;
+  margin-top: 0.5em;
+  &.invisible {
```

Comment:
Is this used?

Reply:
Yep, it is the capslock hidden message.

Reply:
I'm not sure what that is... is it a browser/OS feature that shows you the Caps Lock key is active?

Reply:
Its actually something we built I think. It looks for if caps lock is pressed. The div the message is in is normally invisible, but it still takes up space when not shown. I set it to display non when it is invisible as to not have an awkward gap.

Reply:
Thanks, that makes sense. In that case, our convention is to use `"hidden"` as the class instead of `"invisible"`, that will automatically hide the element without the specific CSS rule.